.github/scripts/get_test_server.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+# Script that determines the test server to run the tests against.
+
+current_branch=$(git rev-parse --abbrev-ref HEAD)
+echo "current_branch: $current_branch"
+
+if [[ $current_branch = release/* ]]; then
+    sdk_version=$(cat setup.cfg | grep current_version | cut -d ' ' -f 3)
+    echo "sdk_version: $sdk_version"
+    IFS=. read -r major minor patch <<< "$sdk_version"
+    sdk_version_major_minor="$major.$minor"
+
+    prod_version=$(curl --silent --max-time 5 "https://cloud.kili-technology.com/api/label/v2/version" | jq -r .version)
+    echo "prod_version: $prod_version"
+    IFS=. read -r major minor patch <<< "$prod_version"
+    prod_version_major_minor="$major.$minor"
+
+    preprod_version=$(curl --silent --max-time 5 "https://preproduction.cloud.kili-technology.com/api/label/v2/version" | jq -r .version)
+    echo "preprod_version: $preprod_version"
```

Comment:
Does getting a version from this URL guaranty that preproduction is up and working? Just wondering the behavior if there is a deployment issue or another one

Reply:
Well no not sure :/  But I guess if preprod is up (but not working), many tests later on should crash?